esp32-hal/Cargo.toml

```diff
@@ -50,7 +50,7 @@ ssd1306            = "0.8.4"
 static_cell        = { version = "2.0.0", features = ["nightly"] }
 
 [features]
-default            = ["rt", "vectored", "xtal-40mhz"]
+default            = ["rt", "vectored", "xtal-40mhz", "embassy-integrated-timers"]
```

Comment:
Sorry if I'm missing something, but it seems a little strange to have this enabled when Embassy support isn't even enabled by default.

Should this not just need selecting in the same way that the executors and time drivers currently do? It's obviously not great DX to need all these features but I'm not sure what the alternative is at this point.

Reply:
I wanted to have some sensible defaults and avoid breaking old code. Previously we had these hardcoded and I think even with the added documentation which one to use is a somewhat hard question.

Reply:
Okay thanks for the explanation, I think this is fine for now at least.